templates/zone.header.erb

```diff
@@ -11,3 +11,5 @@ $TTL <%= @ttl %>
 <% if ! @reverse %>
 <%= @soa %>. IN A <%= @soaip %>
 <% end %>
+
+$INCLUDE <%= @zonefilepath -%>/<%= @filename -%>.static <%= @zone -%>.
```

Comment:
I introduced a `static_records` variable in the `dns::zone` type, which defaults to false. Rspec testing tests both implementations (static_records true as well as false)